traveler-ios-ui/TravelerKitUI/ViewControllers/CatalogItemDetailsViewController.swift

```diff
@@ -7,7 +7,31 @@
 //
 
 import UIKit
+import TravelerKit
 
 class CatalogItemDetailsViewController: UIViewController {
-    
+
+    var itemDetails: CatalogItemDetails?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        switch itemDetails {
+        case let itemDetails as BookingItemDetails:
+            performSegue(withIdentifier: "bookingSegue", sender: itemDetails)
+        default:
+            Log("Unknown item type", data: nil, level: .error)
```

Comment:
+1

Reply:
You can also pass the itemDetails instance as data to the log so it prints it on the console. Usually error logs have some sort of data associated with it to help debug the issue